frontend/public/components/app.jsx

```diff
@@ -303,6 +304,20 @@ const CaptureTelemetry = React.memo(function CaptureTelemetry() {
   return null;
 });
 
+const SetTheme = React.memo(function SetTheme() {
+  const [theme, , themeLoaded] = useUserSettings('console.theme');
+  React.useEffect(() => {
+    const root = document.getElementsByTagName('html')[0];
+
+    if (themeLoaded && theme === 'dark') {
+      root.classList.add('pf-theme-dark');
+    } else {
+      root.classList.remove('pf-theme-dark');
+    }
+  }, [theme, themeLoaded]);
+  return null;
+});
```

Comment:
@Sam @vikram-raj  cc @mattnolting 
Related to this functionality, we use `iframes` to selectively display content within OperatorHub (possibly elsewhere?) This content has is own `<html>` markup structure and will also need the `pf-theme-dark` class added to those, in order to have access to the dark theming css. Current display with `pf-theme-dark` only at `root` `<html>`

<img width="1181" alt="Screen Shot 2022-03-02 at 11 43 41 AM" src="https://user-images.githubusercontent.com/1874151/156407550-d30e773c-cbad-4102-89fa-46c640ec2adc.png">

<img width="1303" alt="Screen Shot 2022-03-02 at 11 27 38 AM" src="https://user-images.githubusercontent.com/1874151/156407089-9d8832b9-8444-436f-afb8-9ef8bc7239e3.png">

Reply:
FYI: our epic is https://issues.redhat.com/browse/ODC-5990 and we try to address just the user preferences here and then will focus on general CSS issues. But it's a good point anyway.

@vikram-raj I believe Iframe support is not one of the easy "fix everything we can find problems". Maybe we can add this as a separate story to don't forget it? :)

Reply:
![Peek 2022-03-03 18-17](https://user-images.githubusercontent.com/2561818/156574765-5729fa25-d3ba-46f3-96cc-2c5fb9cb6cef.gif)

Reply:
@sg00dwin is there a reason you still use markdown in an iframe instead of inline? We have added support for inlining the markdown a while ago.